src/scss/components/password-input/_index.scss

```diff
@@ -0,0 +1,106 @@
+@import "components/form/config";
+@import "mixins";
+
+hx-password-control {
+  $ctrl-columns: auto 1fr auto;
+  $ctrl-rows: auto auto;
+  $ctrl-row-align: center;
+
+  -ms-grid-columns: $ctrl-columns;
+  -ms-grid-rows: $ctrl-rows;
+  align-items: $ctrl-row-align;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "label  label   label"
+    "prefix control suffix";
```

Comment:
### The grid needs to be updated to account for `Help Text` and `Error Message`:

![Screen Shot 2020-03-04 at 11 42 04 AM](https://user-images.githubusercontent.com/10120600/75910115-e6091a80-5e12-11ea-9aa4-2b0cfd711fc7.png)

Reply:
Please see https://github.com/HelixDesignSystem/helix-ui/pull/665#discussion_r398931016 for updated implementation.